src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+
+import exampleFacetedInnerJoin from 'images/nrql_screenshot-crop_example-faceted-inner-join.webp'
+
+import exampleColumnarInnerJoin from 'images/nrql_screenshot-crop_example-columnar-inner-join.webp'
+
+import exampleJoinedPercentile from 'images/nrql_screenshot-crop_example-joined-percentile.webp'
+
+import exampleJoinedMedian from 'images/nrql_screenshot-crop_example-joined-median.webp'
+
+import queriesnrqlApdexNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_apdex-NRQL-query-builder.webp'
+
+import lowerExample from 'images/lowerExample.webp'
+
+import percentile0 from 'images/percentile_0.webp'
+
+import positionExample from 'images/positionExample.webp'
+
+import queriesnrqlPercentileNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_percentile-NRQL-query-builder.webp'
+
+import queriesnrqlFilterNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_filter-NRQL-query-builder.webp'
+
+import screenshotInsightsFilter00 from 'images/screenshot_insights_filter_0_0.webp'
+
+import substringExample from 'images/substringExample.webp'
+
+import upperExample from 'images/upperExample.webp'
+
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) was
+created to query telemetry data.
+
+Every query that is executed only considers telemetry data that falls in a specified time range. If a query request
+does not specify a time range, the last hour of telemetry data will be queried.
+
+## Specifying a time range [#spec-time-range]
+There are two ways to specify the time range:
+
+### Using the dashboard [time picker](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker)
+
+When viewing a dashboard, the time picker time range will supersede a query's `SINCE` and `UNTIL` clauses, unless the widgets has been configured to disable
+the time picker.
+
+### Include [`SINCE`](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions#sel-since) and/or [`UNTIL`](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions#sel-until)
+
+  Simple example: `FROM Foo SELECT * SINCE 2 days ago UNTIL 1 day ago`
+
+  See the [Specifying a time](#spec-time) section below for a list of supported time formats.
+
+### Time range snapping
+Depending on the type of query, the specified `SINCE` and `UNTIL` times may be adjusted at execution time.
+
+<Collapser id="timeseries-snap-example" title="Metric query snapping">
+  Metric events and some sample event types may need to adjust the specified time range to the metric's roll-up time aggregations.
+
+  Example query: `FROM Metric SELECT ... SINCE '2023-12-20T00:04Z' UNTIL '2023-12-22T00:03Z'`
+
+  Looking at the JSON response to this query, this 47 hour and 59 minute time range resolves to 5 minute Metric time
+  aggregations, so the actual time range snaps to nearby aggregation edges,
```

Comment:
aggregation boundaries?

Reply:
That link goes to the "Facet results by time" page - I don't see any thing related to SQL time formatting there. Did you paste the wrong link?

As far as I know, SQL's standard date time format is "YYYY-MM-DD HH:MI:SS", which we do support. Are you saying you want the simple example changed? I could do something like `FROM Foo SELECT * SINCE '2023-12-18 12:34:56' UNTIL 1 day ago`, but I'm not sure I want to mix time reference formats in that simple example, which is really about showing the SINCE and UNTIL construct.

Reply:
I believe he's referring to the "```sql" formatting applied to queries on that page, e.g.

> ```sql
> SELECT count(*) FROM K8sDaemonsetSample FACET monthOf(createdAt)
> ```